### ISSUE TITLE
add enx-redirect tests to E2E

### ISSUE THREAD

**Issue by mariliamelo** [google/exposure-notifications-verification-server#1574]
### TL;DR
Add tests inside E2E to verify enx-redirect is working properly.

### Design

**Proposal**
The [enx-redirect service](https://github.com/google/exposure-notifications-verification-server/blob/main/cmd/enx-redirect/main.go) is one of our core services where users can use it verify the code they have been issued. 
We currently don't have a regular test suite that covers the enx-redirect service functionality on an ongoing basis.
Our [E2E test](https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/clients/e2e.go) simulates a mobile device uploading TEKs and tests the key functionality of the system, but currently it doesn't include any enx-redirect use cases. 
This issue proposes adding enx-redirect tests to E2E in order to improve the observability of enx-redirect service. 

**Alternatives considered**
We considered having an SLO to track the overall availability of the enx-redirect service, but due to skewed traffic to business hours and difficult in implementing different SLOs for different hours with the current monitoring system, we decided to drop the SLO implementation.

**Resources**

- [Enx-redirect documentation](https://github.com/google/exposure-notifications-verification-server/blob/main/docs/link_handling.md)

**Additional information**

**Comment by sethvargo** [comment]
The scaffolding for this is in. Still need to do edge testing and add monitoring/alerting.

**Comment by sethvargo** [comment]
Okay, this is done. The existing CloudSchedulerFailed alert will alert us if it starts failing.
